.github/workflows/release-python.yml

```diff
@@ -29,43 +29,25 @@ jobs:
         # Github Actions doesn't support pairing matrix values together, let's improvise
         # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
         buildplat:
-        - [ubuntu-20.04, manylinux_x86_64]
-        - [ubuntu-20.04, manylinux_aarch64]
-        - [ubuntu-20.04, manylinux_ppc64le]
-        - [ubuntu-20.04, manylinux_s390x]
-        - [ubuntu-20.04, manylinux_i686]
-        - [windows-2019, win_amd64]
-        - [windows-2019, win32]
-        python: ["cp37", "cp38", "cp39", "cp310", "cp311", "cp312"]
-        include:
-            - buildplat: [macos-11, macosx_*]
-              python: "cp37"
-            - buildplat: [macos-11, macosx_*]
-              python: "cp38"
-            - buildplat: [macos-11, macosx_*]
-              python: "cp39"
-            - buildplat: [macos-14, macosx_*]
-              python: "cp310"
-            - buildplat: [macos-14, macosx_*]
-              python: "cp311"
-            - buildplat: [macos-14, macosx_*]
-              python: "cp312"
```

Comment:
Why can we get away with not specifying the python version anymore Is It that the way we build wheels now will automatically create wheels packages for 3.7 - 3.12?

Reply:
Yes, and we verify it later. :)